### ISSUE TITLE
Only edit author's comments

### ISSUE THREAD

**Issue by diogoleitao** [sinfo/eventdeck#333]
As seen on the picture attached, everyone can edit anyone's comments. This should not be possible (I guess).

![bug](https://cloud.githubusercontent.com/assets/5692154/19854650/0074c71a-9f68-11e6-9977-5179898043df.png)

**Comment by JGAntunes** [comment]
Yes, it's true it shouldn't, but right now there's no easy way of dealing
with permissions because some people should be able to edit a shitload of
stuff, others not that much. We're aiming in solving this in a near future
hopefully.

João Antunes
